editor.planx.uk/src/@planx/components/PropertyInformation/Editor.tsx

```diff
@@ -0,0 +1,85 @@
+import { TYPES } from "@planx/components/types";
+import {
+  EditorProps,
+  ICONS,
+  InternalNotes,
+  MoreInformation,
+} from "@planx/components/ui";
+import { useFormik } from "formik";
+import React from "react";
+import Input from "ui/Input";
+import InputRow from "ui/InputRow";
+import ModalSection from "ui/ModalSection";
+import ModalSectionContent from "ui/ModalSectionContent";
+import OptionButton from "ui/OptionButton";
+import RichTextInput from "ui/RichTextInput";
+
+import { parseContent, PropertyInformation } from "./model";
+
+type Props = EditorProps<TYPES.PropertyInformation, PropertyInformation>;
+
+export default PropertyInformation;
+
+function PropertyInformation(props: Props) {
+  const formik = useFormik({
+    initialValues: parseContent(props.node?.data),
+    onSubmit: (newValues) => {
+      props.handleSubmit?.({
+        type: TYPES.PropertyInformation,
+        data: newValues,
+      });
+    },
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit} id="modal">
+      <ModalSection>
+        <ModalSectionContent
+          title="Propery information"
+          Icon={ICONS[TYPES.PropertyInformation]}
+        >
+          <InputRow>
+            <Input
+              format="large"
+              name="title"
+              placeholder={formik.values.title}
+              value={formik.values.title}
+              onChange={formik.handleChange}
+            />
+          </InputRow>
+          <InputRow>
+            <RichTextInput
+              name="description"
+              placeholder="Description"
+              value={formik.values.description}
+              onChange={formik.handleChange}
+            />
+          </InputRow>
+          <OptionButton
+            selected={formik.values.showPropertyTypeOverride}
+            onClick={() => {
+              formik.setFieldValue(
+                "showPropertyTypeOverride",
+                !formik.values.showPropertyTypeOverride
+              );
+            }}
+          >
+            Show users a "change" link to override the property type
+          </OptionButton>
```

Comment:
In practice, I think we'll always use this component with this option toggled "on" but I like the idea of introducing it behind a flag here because it gives us a bit more flexibility in deciding our content migration plan. 

We could do the "simple" migration that inserts it after every FindProperty with the option off to satisfy initial content-parity, and in a separate migration we could toggle it on and insert the whole tree of property type questions in-between.